scripts/poddbmismatch.py

```
@@ -0,0 +1,235 @@
+
+    return {
+        "host": values.get("host", "localhost"),
+        "port": values.get("port", "5432"),
+        "database": values.get("database", ""),
+        "username": values.get("username", ""),
+        "password": values.get("password", ""),
+        "sslmode": values.get("sslmode", "disable"),
+    }
+
+
+def fetch_cluster_pods(context: Optional[str], include_completed: bool) -> Set[str]:
```

Ruff (0.15.12)

[warning] Boolean-typed positional argument in function definition

(FBT001)